static/css/blog.css

```diff
@@ -321,8 +321,7 @@ div.post-footer {
   font-size: 16px;
   line-height: 1.5;
 }
---></style>
```

Comment:
I'm not sure what this was even doing here -- can you blame it and see what commit added it?

Reply:
https://github.com/iap/website/blame/88a76d7632adec440912ffb31e90de19969929b6/static/css/blog.css